src/pages/Product/Product.tsx

```diff
@@ -0,0 +1,20 @@
+import { useParams } from 'react-router-dom';
+import { API } from 'api/products';
+import { useFetch } from 'hooks/useFetch';
+import { Related } from './components/Related';
+import { Showcase } from './components/Showcase';
+import styles from './Product.module.scss';
+
+export function Product() {
```

Comment:
давай все компоненты через стрелочные функции делать

```suggestion
export const Product: React.FC = () => {
```